x-pack/plugin/core/src/main/resources/org/elasticsearch/xpack/entsearch/analytics/behavioral_analytics-events-default_policy.json

```diff
@@ -19,6 +19,10 @@
         }
       }
     },
+    "cold": {
```

Comment:
ℹ️ Adding a cold tier, allowing leveraging cold nodes if it exists into the cluster.